src/includes/performance/distributed-tracing/dart.mdx

```diff
@@ -0,0 +1,19 @@
+To obtain a trace header from the span, use `ISentrySpan#toSentryTrace()` method. Then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
```

Comment:
`ISentrySpan#toSentryTrace()` is the `#` an actual thing?

Reply:
not really, on Java docs, its a convention that makes docs clickable, in Dart its [] but [] is also used in markdown, so I kept the Java way, I could do `ISentrySpan.toSentryTrace()` though, but it'd look like a static method but its not